KMLConfig.cmake.in

```diff
@@ -4,4 +4,6 @@ set(KML_INCLUDE_DIRS
   "${Boost_INCLUDE_DIR}")
 
 set(KML_LIBRARY_DIR  "${libkml_BINARY_DIR}")
-find_library(KML_LIBRARY kml PATHS "${libkml_BINARY_DIR}" NO_DEFAULT_PATH)
+if(EXISTS "@KML_TARGETS_FILE@" AND NOT TARGET kml)
```

Comment:
This is wrong; `KMLConfig.cmake` should look for `KMLTargets.cmake` in the same directory as itself (). It should *not* embed the build directory, as this will be wrong for installs, especially relocatable installs.

Depending on what version of CMake you're willing to require, instead use either `CMAKE_CURRENT_LIST_DIR` or:

    get_filename_component(_SELF_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)